portfolio/src/main/webapp/mapsScript.js

```diff
@@ -12,36 +12,86 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// This function has code that is referenced from the GoogleMaps Platform
+// Documentation: https://developers.google.com/maps/documentation/javascript/tutorial
 
 function initMap() {
     const COORDINATES_WATERLOO = {lat: 43.4643, lng: -80.5204};
     const COORDINATES_TORONTO = {lat:43.6532, lng: -79.3832};
+
     let map = new google.maps.Map(document.getElementById('map'), {
-    zoom: 4,
-    center: COORDINATES_WATERLOO
+        zoom: 8,
+        center: COORDINATES_WATERLOO
     });
 
     const MARKER_WATERLOO = new google.maps.Marker({
-    position: COORDINATES_WATERLOO,
-    map: map,
-    title: 'Waterloo'
+        position: COORDINATES_WATERLOO,
+        map: map,
+        title: 'Waterloo'
+    });
+    const MARKER_TORONTO = new google.maps.Marker({
+        position: COORDINATES_TORONTO,
+        map: map,
+        title: 'Toronto'
     });
-    let MARKER_TORONTO = new google.maps.Marker({
-    position: COORDINATES_TORONTO,
-    map: map,
-    title: 'Toronto'
+    let displayBounds = new google.maps.LatLngBounds();
+    displayBounds.extend(COORDINATES_WATERLOO);
+    displayBounds.extend(COORDINATES_TORONTO);
+    
+    let input = document.getElementById("search-input");
+    let searchBox = new google.maps.places.SearchBox(input);
+    map.controls[google.maps.ControlPosition.TOP_LEFT].push(input);
+
+    //reset bounds if places returned are out of view
```

Comment:
nit: we write full sentences for comments (i.e. capitalize the first letter and period at the end)

Reply:
Fixed!